packages/labs/gen-wrapper-react/tsconfig.json

```diff
@@ -1,7 +1,9 @@
 {
   "compilerOptions": {
-    "target": "es2020",
-    "module": "es2022",
+    "incremental": true,
+    "tsBuildInfoFile": "tsconfig.tsbuildinfo",
```

Comment:
This is the default (though no object to making that explicit).

Reply:
I had a build error in the test files without adding this there... I did it to all configs in the PR.